pre-docs/bolt_running_plans.md

```diff
@@ -80,3 +80,14 @@ default, the `modulepath` includes `modules/` and `site-modules/` directories in
 your module is in `~/src/modules/my_module/` run Bolt with `--modulepath
 ~/src/module`. If you often use the same `modulepath` you can set `modulepath` in
 `bolt.yaml`.
+
+## Packaged plan content
+
+Bolt is packaged with a collection of modules that contain useful plans to support common workflows.
+
+- [aggregate](https://github.com/puppetlabs/bolt/tree/master/modules/aggregate): Aggregate task, script or command results.
```

Comment:
Should this link to the same list on the `bolt_installing_modules` page instead of duplicating? On the one hand we wouldn't have to update in 2 places. On the other having the list here is easier than clicking through to another page.